docker-build/src/test/java/io/quarkus/ts/docker/DockerBuildIT.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.ts.docker;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+
+import com.github.dockerjava.api.model.Image;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnNative;
+import io.quarkus.test.utils.DockerUtils;
+
+@QuarkusScenario
+@DisabledOnNative // in order to avoid extra test time. In my opinion native mode doesn't add value for this scenario
+public class DockerBuildIT {
+
+    private static final String SPACE = "      ";
+    private static final String PADDED_DOCKER_IMG_NAME = "hello-world-app" + SPACE;
+    private static final String DOCKER_IMG_VERSION = "1.0.0";
+
+    @AfterAll
+    public static void tearDown() {
+        DockerUtils.removeImage(PADDED_DOCKER_IMG_NAME, DOCKER_IMG_VERSION);
+    }
+
+    @Test
+    public void buildDockerImage() {
+        Image dockerImg = DockerUtils.getImage("qe/hello-world-app", "1.0.0");
```

Comment:
I suggest to also check for PADDED_<...>_NAME. And definitely to use DOCKER_IMG_VERSION.